Update blob-inventory-how-to.md (#123461)
Azure/azure-content: articles/storage/blobs/blob-inventory-how-to.md

   If you select **Blob**, then under **Blob subtype**, choose the types of blobs that you want to include in your report, and whether to include blob versions and/or snapshots in your inventory report.

   > [!NOTE]

>    > Versions will be available to include in the blob inventory for accounts with the hierarchical namespace feature disabled.

Reviewer: ```suggestion
   > The option to include blob versions appears only for accounts that don't have the hierarchical namespace feature enabled. 
```
